driver/lib/commands/execute/scroll.ts

```diff
@@ -206,3 +204,13 @@ export const scrollIntoView = async (
 
   return await self.executeElementCommand(`scrollIntoView`, elementBase64, args);
 };
+
+
+export const pageBack = async function (this: FlutterDriver): Promise<void> {
```

Comment:
Implementing `driver.back` is a proper way like https://github.com/appium/appium-flutter-driver/blob/main/driver/lib/driver.ts#L165-L172

I'll adjust some implementation after this merge